docs/sources/reference/run.md

```diff
@@ -310,13 +310,22 @@ The operator can also adjust the performance parameters of the
 container:
 
     -m="": Memory limit (format: <number><optional unit>, where unit = b, k, m or g)
+    -memory-swap="": Total memory limit (memory + swap, format: <number><optional unit>, where unit = b, k, m or g)
     -c=0 : CPU shares (relative weight)
 
-The operator can constrain the memory available to a container easily
-with `docker run -m`. If the host supports swap memory, then the `-m`
-memory setting can be larger than physical RAM.
+For memory, we have three ways to use them:
```

Comment:
Yeah, this is rule for cgroup, but libcontainer did some modification, I'll add some notes, thanks.